src/test/java/uk/gov/hmcts/reform/civil/bpmn/CreateClaimSpecTest.java

```diff
@@ -484,17 +484,6 @@ private void completeCaseAssignment(VariableMap variables) {
         );
     }
 
-    public void startBusinessProcess(VariableMap variables) {
```

Comment:
why is this remove?

Reply:
Adding back for that part. I think it is accidentally removed during development